natlang/src/main/java/de/unistuttgart/iaas/amyassist/amy/natlang/DialogHandlerImpl.java

```diff
@@ -96,7 +96,12 @@ public void process(String naturalLanguageText, UUID uuid) {
 			}
 			if (intent.getPartialNLIClass() != null) {
 				Object object = this.serviceLocator.createAndInitialize(intent.getPartialNLIClass());
-				dialog.output(dialog.getIntent().call(object, stringToEntityData));
+				try {
+					dialog.output(dialog.getIntent().call(object, stringToEntityData));
+				} catch (Exception e) {
+					this.logger.warn("Error during plugin execution", e);
+					dialog.output("There is an error in the plugin. please try again.");
```

Comment:
"please try again" does not help in the most cases. You may refer to the logs for details.